src/main/java/org/broadinstitute/hellbender/dev/pipelines/bqsr/CalibrationTablesBuilder.java

```diff
@@ -0,0 +1,93 @@
+
+package org.broadinstitute.hellbender.dev.pipelines.bqsr;
+
+import com.google.api.services.genomics.model.Read;
+import com.google.cloud.genomics.gatk.common.GenomicsConverter;
+
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMRecord;
+import htsjdk.tribble.Feature;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.engine.ReferenceDataSource;
+import org.broadinstitute.hellbender.tools.recalibration.RecalibrationArgumentCollection;
+import org.broadinstitute.hellbender.tools.recalibration.RecalibrationTables;
+import org.broadinstitute.hellbender.dev.tools.walkers.bqsr.BaseRecalibratorUprooted;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+import java.io.File;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/*
+ * Feed me reads and skipIntervals, and I'll get you a RecalibrationTables.
+ *
+ * (OK, yes, I need the reference genome too).
+ *
+ * This class wraps around BaseRecalibratorUprooted, giving it a cozy environment outside of the normal
+ * CommandLine framework. It's also completely independent of Dataflow, so it could be tested outside of that framework.
```

Comment:
Since you admit it could be tested independently, why not add unit tests for it now as part of this pull request? :)